src/components/PostList/index.jsx

```diff
@@ -123,7 +124,7 @@ export default function PostList(props) {
           <select className="postFilter" id="city" onChange={handleCity}>
             <option value="all">{t("postPage.allLabel")}</option>
             {constantAllPosts.map((el) => {
-              return <option value={el.address[0]}>{el.address[0]}</option>;
+              return <option value={el.address}>{el.address}</option>;
```

Comment:
Are theses still working without the [0]

Reply:
Yes we changed the fireBase